src/components/Image.js

```diff
@@ -0,0 +1,15 @@
+import NextImage from "next/image";
+
+// opt-out of image optimization, no-op
+const customLoader = ({ src }) => {
+  return src
+}
+
+export default function Image(props) {
+  return (
+    <NextImage
+      {...props}
+      loader={customLoader}
+    />
+  );
+}
```

Comment:
```suggestion
// opt-out of image optimization, no-op
export default function Image(props) {
  return (
    <NextImage
      {...props}
      loader={({ src }) => src}
    />
  );
}
```
Something like this should work I guess